rtloader/two/two.cpp

```diff
@@ -46,20 +46,20 @@ Two::~Two()
     // For more information on why Py_Finalize() isn't called here please
     // refer to the header file or the doxygen documentation.
     PyEval_RestoreThread(_threadState);
-    free(_pythonHome);
     Py_XDECREF(_baseClass);
 }
 
 void Two::initPythonHome(const char *pythonHome)
 {
-    free(_pythonHome);
+    char *oldPythonHome = _pythonHome;
     if (pythonHome == NULL || strlen(pythonHome) == 0) {
         _pythonHome = _strdup(_defaultPythonHome);
     } else {
         _pythonHome = _strdup(pythonHome);
     }
 
     Py_SetPythonHome(_pythonHome);
```

Comment:
can you add an inline comment here about not freeing `_pythonHome` because `Py_SetPythonHome` requires that?

(same in `three.cpp`)

Reply:
👍 done